web/src/components/molecules/Asset/Asset/AssetBody/previewToolbar.tsx

```diff
@@ -16,21 +17,27 @@ type Props = {
 const PreviewToolbar: React.FC<Props> = ({
   url,
   selectedPreviewType,
+  isModalVisible,
+  isSVG,
   handleCodeSourceClick,
   handleRenderClick,
   handleFullScreen,
-  isModalVisible,
   handleModalCancel,
 }) => {
   return (
     <>
-      {selectedPreviewType === PreviewType.SVG && (
+      {selectedPreviewType === PreviewType.Image && isSVG && (
         <>
           <Button onClick={handleCodeSourceClick}>Source Code</Button>
           <Button onClick={handleRenderClick}>Render</Button>
         </>
       )}
-      <Button type="link" icon={<Icon icon="download" />} size="large" onClick={handleFullScreen} />
+      <Button
+        type="link"
+        icon={<Icon icon="fullscreen" />}
```

Comment:
Just an idea (for another PR only if you want), but could you guys customize the Button atom so that you just pass the icon name as a string, and the Button will using the Icon component if there is an icon string being passed? 
That way you don't have to always import Icons AND Buttons whenever you need a button  with an icon. Keeps things cleaner (and more like reearth-web).
Up to you if you think its helpful like that

Reply:
Good idea, this should be implemented in the future.